sdk/textanalytics/azure-ai-textanalytics/swagger/Update-Codegeneration.ps1

```diff
@@ -0,0 +1 @@
+& (Join-Path $PSScriptRoot ".." ".." ".." ".." eng scripts Invoke-Codegeneration.ps1) -Directory $PSScriptRoot
```

Comment:
@alzimmermsft is this needed before migrating to use "Stream-Style Serialization"

Reply:
Nope, this is another project going on to validate code generation as part of CI, which will lead to automatically updating code generation centrally instead of requiring all SDK owners to do this manually.

Given that, I would still recommend adding that to Text Analytics.